central/src/components/QuestionDashboard.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { makeStyles, Box, Grid } from '@material-ui/core';
+import Button from '@material-ui/core/Button';
+import { IQuestionTemplate } from '@righton/networking';
+import QuestionCard from './QuestionCard';
+
+type QuestionDashboardProps = {
+  questions: IQuestionTemplate[];
+};
+
+export default function QuestionDashboard({
+  questions
+}: QuestionDashboardProps) {
+  const classes = useStyles();
+  console.log("sup");
+  console.log(questions);
```

Comment:
probably only need one of these

Reply:
removed both in a later PR. Also, went through all the apps and pulled any needless console.logs from them as these kept coming up